be1-go/cli/cli.go

```diff
@@ -71,67 +64,19 @@ func (s *ServerConfig) newHub(l *zerolog.Logger) (hub.Hub, error) {
 		s.ServerAddress = fmt.Sprintf("ws://%s:%d/server", s.PublicAddress, s.ServerPort)
 	}
 
-	var point kyber.Point = nil
-	err := ownerKey(s.PublicKey, &point)
-	if err != nil {
-		return nil, err
-	}
-
-	schemaValidator, err := validation.NewSchemaValidator()
-	if err != nil {
-		return nil, err
-	}
-
-	db, err := sqlite.NewSQLite(s.DatabasePath, true)
-	if err != nil {
-		return nil, err
-	}
-
-	database.InitDatabase(&db)
-
-	serverPublicKey, serverSecretKey, err := db.GetServerKeys()
-	if err != nil {
-		serverSecretKey = crypto.Suite.Scalar().Pick(crypto.Suite.RandomStream())
-		serverPublicKey = crypto.Suite.Point().Mul(serverSecretKey, nil)
-
-		err := db.StoreServerKeys(serverPublicKey, serverSecretKey)
-		if err != nil {
-			return nil, err
-		}
-	}
-
-	err = db.StoreFirstRumor()
+	var ownerPubKey kyber.Point = nil
+	err := ownerKey(s.PublicKey, &ownerPubKey)
 	if err != nil {
 		return nil, err
 	}
 
-	utils.InitUtils(schemaValidator)
+	hub, err := hub2.New(s.DatabasePath, ownerPubKey, s.ClientAddress, s.ServerAddress)
```

Comment:
Can you keep the `hub.New` spelling and just rename the variable to `h` or something ?

Reply:
fixed